components/application-configuration/ui/src/main/resources/PhenoTips/LabeledIdentifierConfigurationClass.xml

```diff
@@ -0,0 +1,225 @@
+  <name>LabeledIdentifierConfigurationClass</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <creationDate>1483543661000</creationDate>
+  <parent>PhenoTips.WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <date>1483550409000</date>
+  <contentUpdateDate>1483550409000</contentUpdateDate>
+  <version>1.1</version>
+  <title/>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+  <content>{{include reference="PhenoTips.TabelarDataMacros"/}}
+
+{{velocity output="false"}}
+#set ($displayDocumentName = 'PhenoTips.LabeledIdentifierConfigurationClass')
+#set ($displayDocument = $xwiki.getDocument($displayDocumentName))
+#set ($labeledIdentifierSettings = 'PhenoTips.LabeledIdentifierSettings')
+#set ($globalIdentifierSettings = 'PhenoTips.LabeledIdentifierGlobalSettings')
+
+#set ($targetObj = $doc.getObject($globalIdentifierSettings))
+#if (!$targetObj)
+  #set ($targetObj = $doc.newObject($globalIdentifierSettings))
+  $doc.save('Added external identifier configuration', true)
+#end
+
+$xwiki.ssx.use($displayDocumentName)##
+{{/velocity}}
+
+{{velocity}}
+#if ("$!{request.xaction}" == 'lastentry')
+ #set ($entryClassName = $request.dataClassName)
+  #set ($options = {})
+  #set ($discard = $options.put('propertyName', "$!{request.propertyName}"))
+  #set ($discard = $options.put('propertyValue', "$!{request.propertyValue}"))
+  #set ($discard = $options.put('labels', "$!{request.withLabel}"))
+  #set ($discard = $options.put('mode', 'edit'))
+  #__extradata_displayLastEntry($request.dataClassName $options)
+#else
+{{html clean="false"}}
+&lt;form action="${preferencesDoc.getURL('save')}" method="post"&gt;
+  &lt;div class="hidden"&gt;
+    &lt;input type="hidden" name="form_token" value="${escapetool.xml($services.csrf.token)}" /&gt;
+    &lt;input type="hidden" name="classname" value="${labeledIdentifierSettings}" /&gt;
+    &lt;input type="hidden" name="xredirect" value="${escapetool.xml($doc.getURL($xcontext.action, $request.queryString))}" /&gt;
+  &lt;/div&gt;
+{{/html}}##
+{{html clean="false"}}
+  &lt;div class="hidden"&gt;
+    &lt;input type="hidden" name="form_token" value="${escapetool.xml($services.csrf.token)}" /&gt;
+    &lt;input type="hidden" name="classname" value="${globalIdentifierSettings}" /&gt;
+    &lt;input type="hidden" name="xredirect" value="${escapetool.xml($doc.getURL($xcontext.action, $request.queryString))}" /&gt;
+  &lt;/div&gt;
+  &lt;h2&gt;$services.localization.render('phenotips.navigation.webHome.eidConfig.heading')&lt;/h2&gt;
+{{/html}}
+  #__extradata_displayTable($labeledIdentifierSettings, {'counter' : false, 'labels' : false, 'mode' : 'edit', 'addedDisplaySheet' : $displayDocumentName})
+  $doc.display("AllowOtherEids", "edit", $targetObj) ##
+  $targetObj.getxWikiClass().AllowOtherEids.translatedPrettyName
```

Comment:
For better usability, this should be a label for the checkbox before, so that clicking on the text toggles the checkbox.